yarn-project/end-to-end/src/integration_l1_publisher.test.ts

```diff
@@ -372,10 +372,9 @@ describe('L1Publisher integration', () => {
 
       const newL2ToL1MsgsArray = block.body.txEffects.flatMap(txEffect => txEffect.l2ToL1Msgs);
 
-      // check that values are not in the outbox
-      for (let j = 0; j < newL2ToL1MsgsArray.length; j++) {
-        expect(await outbox.read.contains([newL2ToL1MsgsArray[j].toString()])).toBeFalsy();
-      }
+      const [emptyRoot] = await outbox.read.roots([block.header.globalVariables.blockNumber.toBigInt()]);
+
+      expect(BigInt(emptyRoot)).toStrictEqual(0n);
```

Comment:
This one could use a note. e.g., "check that we have not yet written any roots to this blocknumber" or something like that. At first I got confused about it being 0, but figured that it is because we are only publishing it, soon after below.